hypervisor/common/hypercall.c

```diff
@@ -94,6 +94,38 @@ int32_t hcall_get_api_version(struct acrn_vm *vm, uint64_t param)
 	return ret;
 }
 
+/**
+ * @brief Get basic platform information.
+ *
+ * The function return basic hardware or configurations information
+ * for current platform.
+ *
+ * @param vm Pointer to VM data structure.
+ * @param param GPA pointer to struct hc_platform_info.
+ *
+ * @pre Pointer vm shall point to SOS_VM
+ * @return 0 on success, non-zero on error.
```

Comment:
I believe @junjiemao1 has indicated a preference to list out all return values in separate entries, i.e.:
```
@retval 0 on success
@retval -1 in case of error
```

Reply:
Yes. The template in #1595 recommends a separate `@retval` for each return value case. This generates the doc in a better style.